test/integration/fuzz/sanitizers/main.fmf

```diff
@@ -0,0 +1,20 @@
+summary: Run dbus-broker under sanitizers
+test: ./test.sh
+require:
+    - dbus-broker
+    - dbus-daemon
+    - dfuzzer
+    - expat-devel
+    - gcc
```

Comment:
I think it would be better to build it with clang because it comes with more checks. (Ideally it would be great to build it with both but if it's either gcc or clang I'd pick clang :-))